### PR TITLE
Add clarifying example to security competency

### LINKED ISSUE #81: J->M feedback on security fixes competency
(Breaking #80 out into four separate issues for discussion)

1. `Regularly fixes security issues raised by others/automation in own code`: wording is unclear to me.  Does it mean: 
    - `Regularly fixes security issues raised by others/Regularly fixes automation in own code` or
    - `Regularly fixes security issues raised by others/Regularly fixes security issues raised by automation in own code`?

### PR DESCRIPTION
**Pull request by alicebartlett**
Closes #81

### CODE REVIEW

File: data/competencies.yml [hunk 1]
```diff
@@ -495,9 +497,10 @@
   domain: null
 
 - id: d8651fc0-059b-4c11-929f-e07803f9f8bf
-  summary: Understands and considers other teams' needs
+  summary: Actively seeks the views of other teams to help guide work
```
Comment (rowanmanning) [inline data/competencies.yml]: The changes to this competency are unrelated. Were these supposed to be a separate PR?

Comment (rowanmanning) [reply data/competencies.yml]: I found and approved the separate PR. This may conflict

Comment (alicebartlett) [reply data/competencies.yml]: Doh!
